roles/emby2/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+#########################################################################
+# Title:         Community: Emby2 Role                                    #
+# Author(s):     desimaniac                                             #
+# URL:           https://github.com/cloudbox/cloudbox                   #
+# Docker Image:  emby/embyserver                                        #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: "Set DNS Record on CloudFlare"
+  include_role:
+    name: cloudflare-dns
+  vars:
+    subdomain: emby2
```

Comment:
This needs to be changed to
`    record: emby2`

It seems community uses an older version of the cloudflare-dns task that references "record" instead of the "subdomain" that main cloudbox uses.

Reply:
Updated